src/restApi/createApi.ts

```diff
@@ -1,82 +1,46 @@
-import {
-  Express,
-  NextFunction,
-  Request,
-  RequestHandler,
-  Response,
-} from 'express';
-import {
-  RequestBody,
-  RequestParams,
-  ResponseList,
-  SuccessResponseType,
-} from './util';
+import {Express, NextFunction, Request, RequestHandler, Response} from 'express';
```

Comment:
@DServy would you mind setting up prettier so formatting changes don't show in PR's... 

Please pull the `.prettierrc` from our time-entry service (where this code would have originally been formatted from) and reformat so this gets back to where it was.